Meta/Azure/Serenity.yml

```diff
@@ -1,8 +1,9 @@
 parameters:
   arch: 'i686'
+  coverage: 'OFF'
 
 jobs:
-  - job: 'Serenity_Clang_${{ parameters.arch }}'
+  - job: 'Serenity_Clang_${{ parameters.arch }}_Coverage_${{ parameters.coverage }}'
```

Comment:
Does it really make sense to run this extra leg for every CI? Instead of say, a nightly job? Are we planning on providing coverage numbers (increase/decrease) per PR in the future?

Reply:
I think it would be pretty cool to display on PRs, yes. I didn't think that far ahead yet though.

my current thoughts were we could deploy the static html pages to some website, or, pull the coverage report into sonar cloud in the nightly job. not sure how to make the second one happen if we don't have coverage metrics for at least every master push, since sonar would have to find the report for the commit it's analyzing and it runs async to the main pipeline.

Reply:
I added a nightly-pipelines.yml to Meta/Azure, but we'll need to add a pipeline in Azure once it's in master.